main.py

```diff
@@ -27,6 +27,7 @@
 import selfplay_mcts
 from gtp_wrapper import make_gtp_instance
 from utils import logged_timer as timer
+from utils import _ensure_dir_exists
```

Comment:
make ensure_dir_exists public, not _prefixed

Reply:
Done